### PR TITLE
Adding logo design files

### PR DESCRIPTION
**Pull request by pooja2299**
I tried converting these png files to svg but the quality of image in svg format was getting degraded. So uploading the original png format

### CODE REVIEW

**Review by matuskalas — CHANGES_REQUESTED** [review]
Awesome, thank you so much @pooja2299 🙌🏽

Would you mind please renaming the files to something like "EDAM_logo_Pooja..." or so, so that people don't overwrite each other's work? I'll merge right after that. Million thanks!!! 🙏🏽

**Comment by pooja2299** [comment]
> Awesome, thank you so much @pooja2299 🙌🏽
> 
> Would you mind please renaming the files to something like "EDAM_logo_Pooja..." or so, so that people don't overwrite each other's work? I'll merge right after that. Million thanks!!! 🙏🏽

done :+1:

**Pull request merged**
